src/Assetic/Filter/JSqueezeFilter.php

```diff
@@ -46,7 +59,7 @@ public function filterLoad(AssetInterface $asset)
 
     public function filterDump(AssetInterface $asset)
     {
-        $parser = new \JSqueeze();
+        $parser = new $this->classname();
```

Comment:
If I've got my references right you can't use a variable as just *part* of the class name in this way - in that case you have to do it your way - but you can use a variable as the *whole* class name.